src/main/scala/mesosphere/marathon/state/Instance.scala

```diff
@@ -20,7 +20,8 @@ case class Instance(
     state: InstanceState,
     tasksMap: Map[Task.Id, Task],
     runSpecVersion: Timestamp,
-    reservation: Option[Reservation]) extends MarathonState[Protos.Json, Instance] {
+    reservation: Option[Reservation],
+    role: String) extends MarathonState[Protos.Json, Instance] {
```

Comment:
What do you think about moving the changes for instance out into a different pull request? They seem orthogonal to me.